README.rst

```diff
@@ -1,3 +1,9 @@
+This package is no longer maintained, and no current Enthought Tool Suite projects
+should rely on it. If you encounter projects that still depend on enthought/enaml,
```

Comment:
> no current Enthought Tool Suite projects should rely on it.

What about projects that are not in ETS, say kitchoi has a library that depends on enaml? Should those projects try to unplug themselves too?

Reply:
> What about projects that are not in ETS, say kitchoi has a library that depends on enaml? Should those projects try to unplug themselves too?

That's handled by the second sentence, IIUC.

Reply:
cleaned up the first sentence to just say "no current projects" instead of "no current ETS projects"

Reply:
Thank you! LGTM!